pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -243,32 +204,6 @@ pub mod pallet {
 	pub type RelayerList<T: Config> =
 		StorageDoubleMap<_, Blake2_128Concat, Domain, Blake2_128Concat, DomainAddress, ()>;
 
-	#[pallet::storage]
```

Comment:
Removed these.

Reply:
Just curious, if the storage is totally empty, do we need to remove it then? I think  only `OutboundMessageNonceStore` need to be removed